### PR TITLE
[Functions] Rename contract setConfig methods

### PR DESCRIPTION
**Pull request by justinkaseman**
### Changes
- To keep consistency for OCR, the OCR2Base contract was reverted from `setOCRConfig` to using the naming `setConfig`
- Because of the above, to avoid name clashes, the naming in `Routable` has been changed from `setConfig` -> `updateConfig` and the event from `ConfigSet` -> `ConfigChanged`

### CODE REVIEW

File: contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol [hunk 1]
```diff
@@ -375,7 +375,7 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     if (weiPerUnitLink <= 0) {
       revert InvalidLinkWeiPrice(weiPerUnitLink);
     }
-    // (1e18 juels/link) * (gas/wei) / (wei/link) = juels per wei
+    // (1e18 juels/link) * (wei/gas) / (wei/link) = juels per wei
```
Comment (bolekk) [inline contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol:378]: juels per gas

Comment (justinkaseman) [reply contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol:378]: true

File: contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol [hunk 2]
```diff
@@ -244,12 +243,21 @@ contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions
       return (resultCode, callbackGasCostJuels);
     }
 
+    // Check that the subscription can still afford
+    if (
+      commitment.adminFee + costWithoutFulfillment + (juelsPerGas * SafeCast.toUint96(commitment.callbackGasLimit)) >
+      s_subscriptions[commitment.subscriptionId].balance
+    ) {
+      resultCode = 4; // IFunctionsRouter.FulfillResult.INSUFFICIENT_SUBSCRIPTION_BALANCE
```
Comment (bolekk) [inline contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol:251]: What happens afterwards? Can the funds be unlocked or is the user still charged?

**Review by bolekk — APPROVED** [review]

File: contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol (hunk 2 above)
Comment (justinkaseman) [reply contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol:251]: User is not charged in this scenario and eventually the request will time out.
We could make it that we charge them what we can, but then we would need to decide who gets what of the incomplete funds.
I don't believe this case should ever actually be hit unless something is going wrong, so if it is we need to alert on it.

**Review by bolekk — APPROVED** [review]

**Pull request merged**